infra/modules/monitoring/main.tf

```diff
@@ -66,6 +66,36 @@ resource "aws_cloudwatch_metric_alarm" "high_app_response_time" {
   }
 }
 
+resource "aws_cloudwatch_metric_alarm" "service_errors" {
+  alarm_name          = "${var.service_name}-errors"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = 1
+  metric_name         = "ErrorCount"
+  namespace           = var.service_name
+  period              = 60
+  statistic           = "Sum"
+  threshold           = 0
+  treat_missing_data  = "ignore"
+  alarm_description   = "Alarm for service errors"
+  alarm_actions       = [aws_sns_topic.this.arn]
+  ok_actions          = [aws_sns_topic.this.arn]
+}
+
+
+resource "aws_cloudwatch_log_metric_filter" "service_error_filter" {
+
+  name           = "service-error-filter"
+  pattern        = "%ERROR%"                     # pattern can find events in unstructured logs
```

Comment:
⭐⭐  &nbsp; _(optional but seriously consider)_

Nice! It this a regex? Can we update it to (ERROR|Exception)?

I feel like `exception` is more commonly going to be in the message than `error` is. In python, at least

Reply:
It is! I found [this](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html#regex-expressions) which was super helpful